src/device.h

```diff
@@ -20,7 +20,7 @@ enum capabilities {
 /** @brief Flags for battery status
  */
 enum battery_status {
-    BATTERY_LOADING         = 65535
+    BATTERY_CHARGING         = 65535
```

Comment:
Such changes are better done in a seperate commit/PR

Reply:
Ah I just saw that it's done seperatly and the rest is squashed. Then it's okay in this case